lib/src/ast/sass/statement/silent_comment.dart

```diff
@@ -12,11 +13,49 @@ class SilentComment implements Statement {
   /// The text of this comment, including comment characters.
   final String text;
 
+  /// The text of this comment that are marked as a documentation comment
+  /// (leading triple slashes).
+  ///
+  /// Provided with slashes and common leading whitespace removed.
```

Comment:
"The subset of lines in `text` that are marked as part of the documentation comments by beginning with `///`.

The leading slashes and first space on each line is removed."

Reply:
Done.